src/views/Editor.vue

```diff
@@ -498,7 +531,7 @@ export default {
 	async mounted() {
 		if (this.id === 'new') {
 			this.clearRecord()
-		} else if (this.id !== 'edit' && this.$store.state.metadata.id !== this.id) {
+		} else if (this.id !== 'edit') {
```

Comment:
This caused issue when user navigated to Datasets view and returned back to the same dataset, the UI was not in sync anymore.

Reply:
Removing this causes unsaved changes to be lost when you go to the Datasets view and return back.

Reply:
I think that we can live with that, what we should do is to add a dialog which warns the user that all unsaved changes are lost

Reply:
the dialog has been added